management/guardian.go

```diff
@@ -11,6 +11,16 @@ type MultiFactor struct {
 	TrialExpired *bool `json:"trial_expired,omitempty"`
 }
 
+type MultiFactorPolicies []string
+
+type MultiFactorProvider struct {
+	//One of auth0|twilio|phone-message-hook
+	Provider *string `json:"provider,omitempty"`
+}
+
+type MultiFactorMessageTypes struct {
+	MessageTypes *[]string `json:"message_types,omitempty"`
+}
```

Comment:
This is okay, but I'm wondering if we should apply the simplification we did in user blocks (see below). Do we expect any other fields to be present in the future for `MultiFactorMessageTypes`? If no we could return a `[]string` from `MessageType()` and its siblings. WDYT?


https://github.com/go-auth0/auth0/blob/40ec8934df33d71cda13fde1360be68fd09f1742/management/user.go#L225-L232
https://github.com/go-auth0/auth0/blob/40ec8934df33d71cda13fde1360be68fd09f1742/management/user.go#L399-L403

Reply:
Hi! I'm sorry but I don't understand how this becomes a simplification. But to answer your question `MultiFactorMessageTypes` would not be expected to contain any other fields.  I don't think it has any siblings, maybe the confusion lies in my name. I changed this in the latest commit.